website/site/content/docs/widgets.md

```diff
@@ -27,310 +27,4 @@ The following options are available on all fields:
-
-- **Name:** `relation`
-- **UI:** text input with search result dropdown
-- **Data type:** data type of the value pulled from the related collection item
-- **Options:**
-  - `default`: accepts any widget data type; defaults to an empty string
-  - `collection`: (**required**) name of the collection being referenced (string)
-  - `searchFields`: (**required**) list of one or more names of fields in the referenced collection to search for the typed value
-  - `valueField`: (**required**) name of the field from the referenced collection whose value will be stored for the relation
-- **Example** (assuming a separate "authors" collection with "name" and "twitterHandle" fields):
-
-  ```yaml
-  - label: Post Author
-    name: author
-    widget: relation
-    collection: authors
-    searchFields: [name, twitterHandle]
-    valueField: name
-  ```
-  The generated UI input will search the authors collection by name and twitterHandle as the user types. On selection, the author name will be saved for the field.
-
-
-## Select
-
-The select widget allows you to pick a single string value from a dropdown menu.
-
-- **Name:** `select`
-- **UI:** HTML select input
-- **Data type:** string
-- **Options:**
-  - `default`: accepts a string; defaults to an empty string
-  - `options`: (**required**) a list of options for the dropdown menu; can be listed in two ways:
-      - string values: the label displayed in the dropdown is the value saved in the file
-      - object with `label` and `value` fields: the label displays in the dropdown; the value is saved in the file
-- **Example** (options as strings):
-
-  ```yaml
-  - label: "Align Content"
-    name: "align"
-    widget: "select"
-    options: ["left", "center", "right"]
-  ```
-- **Example** (options as objects):
-
-  ```yaml
-  - label: "City"
-    name: "airport-code"
-    widget: "select"
-    options:
-      - { label: "Chicago", value: "ORD" }
-      - { label: "Paris", value: "CDG" }
-      - { label: "Tokyo", value: "HND" }
-  ```
-
-
-## String
-
-The string widget translates a basic text input to a string value. For larger textarea inputs, use the [text](#text) widget.
-
-- **Name:** `string`
-- **UI:** text input
-- **Data type:** string
-- **Options:**
-  - `default`: accepts a string; defaults to an empty string
-- **Example:**
-
-  ```yaml
-  - {label: "Title", name: "title", widget: "string"}
-  ```
-
-
-## Text
-
-The text widget takes a multiline text field and saves it as a string. For shorter text inputs, use the [string](#string) widget.
-
-- **Name:** `text`
-- **UI:** HTML textarea
-- **Data type:** string
-- **Options:**
-  - `default`: accepts a string; defaults to an empty string
-- **Example:**
-
-  ```yaml
-  - {label: "Description", name: "description", widget: "text"}
-  ```
+## Default widgets
```

Comment:
I wonder about using "Included widgets" or something like that instead of "Default".